### PR TITLE
Rename propertyId to configId.

### PR DESCRIPTION
**Pull request by jfkhoury**
There is no concept of Property ID as of yet. It's really the config ID that will be provided when you configure your property in the Configuration Service UI.

https://jira.corp.adobe.com/browse/CORE-35789

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All tests pass and I've made any necessary test changes.
- [ x ] I've updated the schema in extension.json or no changes are necessary.

### CODE REVIEW

File: src/lib/runAlloy.js [hunk 1]
```diff
@@ -2242,9 +2242,9 @@ governing permissions and limitations under the License.
         return function (config) {
           var componentRegistry = createComponentRegistry();
           var imsOrgId = config.imsOrgId,
-            propertyId = config.propertyId,
+            configId = config.configId,
             cookieDomain = config.cookieDomain;
-          var cookieName = ALLOY_COOKIE_NAME + "_" + propertyId;
+          var cookieName = ALLOY_COOKIE_NAME + "_" + configId;
```
Comment (jfkhoury) [inline src/lib/runAlloy.js]: Hey @Aaronius, I think we need to update this to use the new Cookie naming structure?

File: src/view/configuration/configuration.jsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ const contextOptions = ["web", "device", "environment", "placeContext"];
 
 const getInstanceDefaults = initInfo => ({
   name: "alloy",
-  propertyId: "",
+  configId: "",
   imsOrgId: initInfo.company.orgId,
   edgeDomain: "alpha.konductor.adobedc.net",
```
Comment (jfkhoury) [inline src/view/configuration/configuration.jsx]: We are currently hitting `beta.adobedc.net`. Should we update it here now, or wait for the 3rd party domain changes that are coming?

File: src/lib/runAlloy.js (hunk 1 above)
Comment (Aaronius) [reply src/lib/runAlloy.js]: The contents of `runAlloy.js` is currently manually copied in from the alloy build. In other words, you should run `npm run build:prod` in Alloy, then copy the contents of `dist/reactor/alloy.js` from Alloy and paste it into this `src/lib/runAlloy.js` file (see the comments for where to paste it). Is this answering your question?

File: src/view/configuration/configuration.jsx (hunk 2 above)
Comment (Aaronius) [reply src/view/configuration/configuration.jsx]: Changing it here would be good.

**Comment by Aaronius** [comment]
👍

**Pull request merged**